### DISCUSSION TITLE
Typo in Spanish translation of iOS app

### DISCUSSION THREAD

**Discussion by oxixes** [community/community#11142] [Mobile] [answered]
There is a typo in GitHub for iOS as shown in this picture, it should be “actualizado” just as it’s shown in the next line.
![d903f6daf268c2e9b5763573836effaf45286029_2_689x481](https://user-images.githubusercontent.com/38050447/152866717-5ea3f45f-5c8b-41c9-bbe3-33cda3a5e134.jpeg)

**Comment by mxie** [comment, marked as answer]
Thanks for letting us know @oxixes! We'll fix it ASAP.
